src/BlazorDatasheet/Menu/SheetMenuOptions.cs

```diff
@@ -22,6 +22,11 @@ public class SheetMenuOptions
     /// </summary>
     public bool MergeEnabled { get; set; } = true;
 
+    /// <summary>
+    /// Enables the menu "UnMerge" option
+    /// </summary>
+    public bool UnMergeEnabled { get; set; } = true;
```

Comment:
I think this could just be shared with MergeEnabled rather than adding another option to the menu options.